xray/run.py

```diff
@@ -104,6 +110,9 @@ def main():
     parser.add_argument("--count")
     parser.add_argument("--nobuild-neptun", action="store_true")
     parser.add_argument("--nobuild-xray", action="store_true")
+    parser.add_argument("--save-output", action="store_true")
+    parser.add_argument("--disable-drop-privileges", action="store_true")
+    parser.add_argument("--ascii", action="store_true")
```

Comment:
I think it might be better to have this take a value and then create an enum for it where the options could be `Ascii` and `Graphical` (or something like that) for now. I don't see a value in outputting both formats at the same time. It would also make it slightly easier to understand in the analyzer script if it was `if output_type == OutputType.Ascii:` instead of just `if ascii:`

Reply:
Are we anticipating to have more than two, `Ascii` and `Graphical` output types to justify the additional boiler plate code?

Reply:
I thought it was quite convenient to have a more detailed graphical view available to investigate the result, but if we don't see cases that that will be useful I will remove it.

Reply:
You're right, it's probably overkill for now, we can keep this as it is :smile: